### ISSUE TITLE
losing focus from default side menu icon after watching a movie/show

### ISSUE THREAD

**Issue by vella9** [jurialmunkey/skin.arctic.fuse#655]
### Skin section

Widgets

### Current Behavior

Hi jurial, sorry to bother you again, but issue #649 resurfaced. Focus is getting lost when moving from first side menu icon [in my case TV Shows, which is set to default] to the widget row again, this time under a different scenario. I am using classic mode and I am on latest version 0.6.5. I also have Navigation>Content>Move between widget groups set to disabled.

This time, it is occurring once you return to the main screen after watching an movie/episode.
Once you return to the main screen, any button press will take you to the first side menu item [TV Shows which is set to default]. From there, you can already notice that the first widget row is "darkened" and if you press the right button, focus is lost. A reload skin only will solve this issue.

### Expected Behavior

once right button is pressed from first side menu icon, after watching a movie/episode, focus should be on the first item of the first widget.

### Steps To Reproduce

watch a movie/episode
once you are returned to the main screen, press any button and this will take you to the first side menu item
press right to focus on first item of first widget
focus is lost

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Do you have the `Autoclose OSD after X seconds` setting enabled? Is this after stopping a video, or just after playback ends?

**Comment by vella9** [comment]
Yes I have that setting enabled.

I can confirm that this is happening when I press the stop button.

When playback ends it does not happen.

**Comment by jurialmunkey** [comment]
In the case, I'm fairly sure it will be fixed by this commit which will be in next update: https://github.com/jurialmunkey/skin.arctic.fuse/commit/69d564b7c0a826a9582a359d2fdc791fd2ff5a2b

I'll close for now but if it is still happening after next update (0.6.7) then let me know here and I'll reopen

**Comment by vella9** [comment]
can confirm that this is solved in 0.6.7.
thanks again 🙏

**Comment by vella9** [comment]
Hi @jurialmunkey , today this happened again on 0.6.7.

I was watching an episode, and when credits started to roll, I paused it and used the next button from the video osd to play the next episode. I watched the second episode and once the credits started to roll of the second episode, I paused it and then pressed the stop button and this triggered the issue again.

sorry to bother you again on this one. 🙏
